docs/source/index.rst

```diff
@@ -238,8 +238,8 @@ We can make use of the `ADS module <https://ads.readthedocs.io/en/latest/>`_ to
 For example we could get the reference for the orbital period of J0737-3039A with
 
     >>> from psrqpy import QueryATNF
-    >>> query = QueryATNF(params='PB', psrs='J0737-3039A', include_refs=True, adsref=True)
-    >>> print(query.table['PB_REFURL'][0])
+    >>> query = QueryATNF(params='PB', psrs='J0737-3039A', adsref=True)
```

Comment:
Doesn't this still need the `include_refs` argument? It doesn't appear to work right now

Reply:
Ah, yes. I briefly removed the `include_refs` options during testing, but then put it back in without re-fixing the docs. I've done that now in c00b4f96f3b